python/ray/serve/_private/http_state.py

```diff
@@ -191,6 +191,7 @@ def update(self):
                     f"{self._node_id} after {DEFAULT_HEALTH_CHECK_TIMEOUT_S}s"
                 )
                 self.try_update_status(HTTPProxyStatus.UNHEALTHY)
+            return
```

Comment:
prefer to put this in an explicit `else` block and please add a comment about why this early return is necessary